sigh/lib/assets/resign.sh

```diff
@@ -201,8 +201,9 @@ then
 fi
 
 # Set the app name
-# The app name is the only file within the Payload directory
-APP_NAME=$(ls "$TEMP_DIR/Payload/")
+# In Payload directory may be another file except .app file, such as StoreKit folder.
+# Search the first .app file within the Payload directory
+APP_NAME=$(ls "$TEMP_DIR/Payload/"|grep ".app$"| head -1)
```

Comment:
@Acorld I could see the problem happening if for whatever reason the `resign.sh` script was called with a "Something.app" file instead of directory as ORIGINAL_FILE argument. Or if the ipa was mangled in some way. I don't like assumptions not being verified :)